frontend/src/components/common/form/ParentSelect.tsx

```diff
@@ -1,10 +1,11 @@
-import { getIn, useFormikContext, setIn } from 'formik';
+import { getIn, useFormikContext } from 'formik';
```

Comment:
These changes fix the ParentSelect component such that it no longer sets agencyId to an object.

Reply:
Woo, thank you!